File: examples/js/scrape_example.js
// Structured Web Scrape via SDK (GET)
const Qoery = require('qoery');

const client = Qoery.ApiClient.instance;
const apiKeyAuth = client.authentications['ApiKeyAuth'];
apiKeyAuth.apiKey = 'YOUR_API_KEY';

const scrapingApi = new Qoery.WebScrapingApi();

const url = 'https://example.com/statistics';
const opts = {
  paragraphExtraction: false,
  plot2table: 0
};

scrapingApi.scrapeGet(url, opts, (error, data) => {
  if (error) {
    throw error;
  }
  console.log(data);
});

File: examples/python/scrape_example.py
import qoery

configuration = qoery.Configuration()
configuration.api_key['ApiKeyAuth'] = 'YOUR_API_KEY'

with qoery.ApiClient(configuration) as api_client:
    api = qoery.WebScrapingApi(api_client)
    data = api.scrape_get(
        url="https://example.com/statistics",
        paragraph_extraction=False,
        plot2table=0,
    )
    print(data)

